Triphouse-project/index.html

```diff
@@ -60,6 +60,35 @@ <h1>
               </div>
             </div>
           </div>
+          <div class="header__mob">
+            <div class="header__menu_mob">
+                <div class="header__logo_mob">
+                    <img src="../img/logo-footer.png" alt="Triphouse"/>
+                </div>
+                <div class="header__buttons_mob">
+                    <a href="#">
+                        <img src="../img/Night_mob.png" alt="Night"/>
+                    </a>
+                    <a href="#">
+                        <img src="../img/AccountCircle_mob.png" alt="Account"/>
+                    </a>
+                    <a href="#">
+                        <img src="../img/menu.png" alt="Menu"/>
+                    </a>
+                </div>
+            </div>
+            <div class="header__search_mob">
+                <p class="header__title_mobile">Discover stays<br/>
+                    to live, work or just relax</p>
+            </div>
+            <div class="header__download_mob">
+                <a href="#">
+                    <img src="./img/google-play-badge.svg" alt="Google Play"/>
+                </a>
+                <a href="#">
+                    <img src="./img/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917 1.svg" alt="App Store"/>
```

Comment:
The name of the file shout be short and without spaces. Check all file and rename.